### DISCUSSION TITLE
Upgrade 2.11.3 to 3.0.0

### DISCUSSION THREAD

**Discussion by suncruise** [netbox-community/netbox-docker#763] [Getting Help]
Hi, I am unable to upgrade form 2.11.3 to 3.0.0. We use the version variable option for upgrades. The the docker-compose file is set as follows:
!
image: netboxcommunity/netbox:${VERSION-v3.0.0}
!
docker-compose down
docker-compose pull
docker-compose up
The result is that the version is still running 2.11.3. I tried the same approach but for 2.11.12 with the same result. Any thoughts?

**Comment by kkthxbye-code** [comment]
You are chaning the default value. You probably have the VERSION environment variable set in some other way. In the file´.env´for example.

You can do `docker-compose config` to see the final compose file that is executed after variable substitutions.

**Comment by suncruise** [comment]
Thanks for your response. You were correct that that we had a version set in the override file, I removed the image statement as it was related to ldap. I then test access to ensure the image came up using version 2.11.3 with no ldap image all good. I then changed the image in the compose file to be netboxcommunity/netbox:${VERSION-v3.0.0} and we now get the following issue after the pull request. 

netbox_1         | ▶️  Running the startup script /opt/netbox/startup_scripts/310_secret_roles.py
netbox_1         | Traceback (most recent call last):
netbox_1         |   File "/opt/netbox/netbox/./manage.py", line 10, in <module>
netbox_1         |     execute_from_command_line(sys.argv)
netbox_1         |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
netbox_1         |     utility.execute()
netbox_1         |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
netbox_1         |     self.fetch_command(subcommand).run_from_argv(self.argv)
netbox_1         |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
netbox_1         |     self.execute(*args, **cmd_options)
netbox_1         |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute
netbox_1         |     output = self.handle(*args, **options)
netbox_1         |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/commands/shell.py", line 93, in handle
netbox_1         |     exec(sys.stdin.read(), globals())
netbox_1         |   File "<string>", line 1, in <module>
netbox_1         |   File "/usr/lib/python3.9/runpy.py", line 285, in run_path
netbox_1         |     return _run_code(code, mod_globals, init_globals,
netbox_1         |   File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
netbox_1         |     exec(code, run_globals)
netbox_1         |   File "/opt/netbox/netbox/../startup_scripts/__main__.py", line 27, in <module>
netbox_1         |     runpy.run_path(f.path)
netbox_1         |   File "/usr/lib/python3.9/runpy.py", line 268, in run_path
netbox_1         |     return _run_module_code(code, init_globals, run_name,
netbox_1         |   File "/usr/lib/python3.9/runpy.py", line 97, in _run_module_code
netbox_1         |     _run_code(code, mod_globals, init_globals,
netbox_1         |   File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
netbox_1         |     exec(code, run_globals)
netbox_1         |   File "/opt/netbox/startup_scripts/310_secret_roles.py", line 2, in <module>
netbox_1         |     from secrets.models import SecretRole
netbox_1         | ModuleNotFoundError: No module named 'secrets.models'; 'secrets' is not a package
netbox-test_netbox_1 exited with code 1
